prisma/schema.prisma

```diff
@@ -71,3 +72,35 @@ model VerificationToken {
 
     @@unique([identifier, token])
 }
+
+model Book {
+    id          Int       @id @default(autoincrement())
+    title       String
+    author      String
+    createdAt   DateTime  @default(now())
+    updatedAt   DateTime  @updatedAt
+    createdBy   User      @relation(fields: [createdById], references: [id])
+    createdById String
+    chapters    Chapter[]
+}
+
+model Chapter {
+    id        Int       @id @default(autoincrement())
+    title     String
+    content   String
+    createdAt DateTime  @default(now())
```

Comment:
nit: usually, createdAt and updatedAt are grouped with id, and there is a space between updatedAt and the rest of hte fieilds. makes it easier to read